.github/workflows/workflow.yaml

```diff
@@ -1,7 +1,11 @@
 name: Continuous integration
 on:
-  - push
-  - pull_request
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Possibly only for my enlightenment: Why only pull requests to master?

Reply:
Fixed in 648604a061c0b6e6b8ebe9c3e91b2a6c49fc6e49